tests/integration/project/cmd_project_test.go

```diff
@@ -26,6 +26,8 @@ var _ = Describe("odo project command tests", func() {
 	// Clean up after the test
 	// This is run after every Spec (It)
 	var _ = AfterEach(func() {
+		// remove later
+		helper.CmdRunner("odo", "project", "list", "-o", "json")
```

Comment:
Remove it

Reply:
this is for debugging, the tests are still failing

Reply:
ok

Reply:
@mik-dass pls mark this PR as work in progress if you plan to do any changes

Reply:
Done